src/enstore_plotter_module.py

```diff
@@ -12,30 +12,76 @@
 import time
 
 # enstore imports
-#import enstore_plotter_framework
 
 class EnstorePlotterModule:
+    """Base class for all Enstore plotter modules."""
     def __init__(self,name,isActive=True):
         self.name=name
         self.is_active=isActive
         self.parameters = {}
+    
     def isActive(self):
```

Comment:
isActive and setActive seem to do real things, is it correct to say they're implemented by subclasses?

Reply:
Good catch.  Thank you.